tubular/edx_api.py

```diff
@@ -517,3 +517,18 @@ def retire_learner(self, learner):
         except HttpDoesNotExistException:
             LOG.info("No license manager data found for user")
             return True
+
+
+class CommerceCoordinatorApi(BaseApiClient):
+    """
+    Commerce-Coordinator API client.
+    """
+    @_retry_lms_api()
+    def retire_learner(self, learner):
+        """
+        Performs the learner retirement step for Commerce-Coordinator.
+        Passes the learner's LMS User Id instead of username.
+        """
+        data = {'edx_lms_user_id': learner['user']['id']}
+        api_url = self.get_api_url('lms/user_retirement')
```

Comment:
Do you plan to add the `lms/user_retirement` endpoint to the commerce-coordinator service? It currently doesn't seem to exist in the django urls.py routes: https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/lms/urls.py

Reply:
Yup! Will be merged in along with this work: https://github.com/edx/commerce-coordinator/pull/247/files#diff-d76063092b846765b0a4c6b15b4a31f3c0bad316aa7e39242bf7d9fcd422a088R19